pkg/backupdriver/backup_repository.go

```diff
@@ -172,3 +212,60 @@ func patchBackupRepositoryClaim(backupRepositoryClaim *backupdriverv1.BackupRepo
 	return nil
 }
 
+// Patch the BackupRepository with the BackupRepositoryClaim name.
+func patchBackupRepository(backupRepository *backupdriverv1.BackupRepository,
```

Comment:
Can you make both patchBackupRepositoryClaim and patchBackupRepository exported functions so I can use them in backupdriver?